shadertoy/OpenGL.cpp

```diff
@@ -395,6 +395,16 @@ class RenderPass final {
                     VertexCubeMap{ ImVec2{ base.x + size.x, base.y + size.y }, ImVec2{ uniformSize.x, 0.0 },
                                    cubeMapVertexPos[cubeMapVertexIndex[idx][3]] },  // right-bottom
                 };
+
+                // For Y-flipped cubemaps, also flip Y coordinates of all faces
+                // to match ShaderToy's UNPACK_FLIP_Y_WEBGL behavior
+                if(mType == NodeType::CubeMapFlippedY) {
```

Comment:
I commented out the Y-flipping logic and the result is correct:
<img width="1492" height="993" alt="image" src="https://github.com/user-attachments/assets/23c83397-352c-466e-a019-7f9f3bea70c4" />

I think the changes to cube vertex indices are enough.